app/controllers/candidate_interface/sign_up_controller.rb

```diff
@@ -5,13 +5,20 @@ def new
     end
 
     def create
-      @candidate = Candidate.new(candidate_params)
+      @candidate = Candidate.find_by(candidate_params)
 
-      if @candidate.save
-        MagicLinkSignUp.call(candidate: @candidate)
-        render :show
+      if @candidate
```

Comment:
that's correct, should we use, for consistency, `@candidate.present?` 
We are already using `.present?`  in various places.

Reply:
I was planning on going with Theo's alternative implementation (see https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/248#discussion_r330518770). What do you think of this way?

Reply:
Conversation continued in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/248#issuecomment-537499668.